code/js/rcalc-math.js

```diff
@@ -30,10 +30,11 @@ function clearDisplay() {
   enableButtons();
 }
 
+// evaluates cotent of main display
 function evaluateDisplay() {
   var valueInserted = nameCalculator.nameDisplayMain.value;
   disableButtons();
-  document.getElementById(ID_DISPLAY_INFO).value = MESSAGE_ERROR_INVALID_ENTRY;
+  document.getElementById(ID_DISPLAY_INFO).value = $.i18n('rcalc-messageErrorInvalidEntry');
```

Comment:
For rcalc-messageErrorInvalidEntry create new files with constants of keys of translations.

Reply:
new file called rcalc-constant-string-key-localization.js has been created and constants of localization keys have been added